syntax = "proto3";

package machina.ifrt_serving;

import "machina/compiler/tf2xla/host_compute_metadata.proto";
import "machina/xla/service/hlo.proto";
import "machina/core/framework/types.proto";
import "machina/core/protobuf/tpu/compile_metadata.proto";

message Tf2HLOResultProto {
  xla.HloModuleProto hlo_module_proto = 1;
  machina.tpu.TPUCompileMetadataProto compile_metadata = 2;
  machina.tf2xla.HostComputeMetadata host_compute_metadata = 3;
}
